modules/mono/editor/Godot.NET.Sdk/Godot.NET.Sdk/Sdk/Sdk.props

```
@@ -81,6 +81,9 @@
       GODOT4_1;GODOT4_1_OR_GREATER;GODOT4_0_OR_GREATER;
       GODOT4_1_0;GODOT4_1_0_OR_GREATER;
     </GodotDefineConstants>
+    <!-- Ensure the define constants don't contain whitespace (see https://github.com/dotnet/roslyn/issues/58391). -->
+    <GodotDefineConstants>$(GodotDefineConstants.Replace('
+','').Replace(' ',''))</GodotDefineConstants>
```

```suggestion
    <GodotDefineConstants>$(GodotDefineConstants.Replace('%0A','').Replace('%0D','').Replace('%09','').Replace(' ',''))</GodotDefineConstants>
```
(not tested) since the linked issue also mentions tabs, maybe replace a few more whitespace chars?